### ISSUE TITLE
[gateway] platform policies should not be executed first on response

### ISSUE THREAD

**Issue by ytvnr** [gravitee-io/issues#7138]
## :collision: Describe the bug

Currently, policies declared at platform level are executed first in the response phase.

They should be executed last: Platform Policies are meant to encompass / wrap the api flows.

Also, don't forget that ON_XXX request are executed before ON_XXX_CONTENT. 

## :sunrise_over_mountains: To Reproduce
Steps to reproduce the behaviour:
1. Configure Policies on API and also on Platform level
2. Use debug mode to see the effective order

## :rainbow: Expected behaviour
Basically, a classic chain would be

PLATFORM POLICIES ON_REQUEST -> API LEVEL POLICIES ON_REQUEST -> PLATFORM POLICIES ON_REQUEST_CONTENT -> API LEVEL POLICIES ON_REQUEST_CONTENT PLATFORM POLICIES ON_RESPONSE -> API LEVEL POLICIES ON_RESPONSE -> PLATFORM POLICIES ON_RESPONSE_CONTENT -> API LEVEL POLICIES ON_RESPONSE_CONTENT

**Comment by LiliaEn** [comment]
Tested on the 3.16.0 env
Browser: Chrome Version 98.0.4758.80 , macOS Big Sur v11.6
Works as expected.

**Comment by gaetanmaisse** [comment]
https://github.com/gravitee-io/release/pull/444
